packages/core-utils/src/itinerary.js

```diff
@@ -488,6 +488,57 @@ export function calculatePhysicalActivity(itinerary) {
   };
 }
 
+const CARBON_INTENSITY_DEFAULTS = {
```

Comment:
This PR is blocked until the other one is merged right? so this change goes away